Python/optimizer.c

```diff
@@ -1234,7 +1238,6 @@ make_executor_from_uops(_PyUOpInstruction *buffer, int length, const _PyBloomFil
         return NULL;
```

Comment:
Note to reviewers: the bad dealloc happens right above this line at line 1233/1238 depending on which diff you view.